net-irc/emech/files/emech-3.0.99_p3-buildfix.patch

```diff
@@ -0,0 +1,70 @@
+diff -Naur emech-3.0.99p3.org/src/gencmd.c emech-3.0.99p3/src/gencmd.c
+--- emech-3.0.99p3.org/src/gencmd.c	2009-05-13 08:51:09.000000000 -0000
++++ emech-3.0.99p3/src/gencmd.c	2018-03-16 22:52:42.591033854 -0000
+@@ -316,7 +316,7 @@
+ 			}
+ 			if (pass == __struct_acces)
+ 			{
+-				printf("\t%i,\t/""* %s *""/\n",
++				printf("\t%li,\t/""* %s *""/\n",
+ 					pre_mcmd[wh].flags & CLEVEL,
+ 					pt);
+ 			}
+@@ -390,7 +390,7 @@
+ 		}
+ 		pass--;
+ 	}
+-	printf("#define LOCALHOST_ULONG %lu\n",inet_addr("127.1"));
++	printf("#define LOCALHOST_ULONG %u\n",inet_addr("127.1"));
```

Comment:
This doesn't look like a correct fix. For fixed-length integers use the appropriate `inttypes.h` macros (`PRI`*). Otherwise you're fixing it on one arch and breaking on the other.